include/aspect/utilities.h

```diff
@@ -45,6 +45,7 @@ namespace aspect
   namespace Utilities
   {
     using namespace dealii;
+    using namespace dealii::Utilities;
```

Comment:
I think @tjhei opposed and I can understand his argument. `dealii::Utilities` contains different utility functions than `aspect::Utilities`. My point was mostly that they are both *utility* functions, and I do not want to remember, which function was in which namespace and would like to avoid the need to specify `aspect::Utilities` vs `dealii::Utilities` every time I use a utility function. I mostly included this for laziness, so if you think we should stick with the old way, I can remove it again.